core/controllers/skill_editor_test.py

```diff
@@ -296,3 +297,49 @@ def test_skill_publish_handler_fails(self):
                     self.url, {'version': 1}, csrf_token=csrf_token,
                     expected_status_int=401)
         self.logout()
+
+
+class SkillEditorQuestionHandlerTests(BaseSkillEditorControllerTests):
+
+    def test_handler_with_new_structure_editors_disabled_raises_error_404(self):
+        self.login(self.ADMIN_EMAIL)
+
+        with self.swap(constants, 'ENABLE_NEW_STRUCTURE_EDITORS', False):
+            self.get_json(
+                '%s/%s' % (feconf.SKILL_EDITOR_QUESTION_URL, self.skill_id),
+                expected_status_int=404)
+
+        self.logout()
+
+    def test_skill_editor_question_handler_updates_question_summary_dicts(self):
+        self.login(self.ADMIN_EMAIL)
+
+        with self.swap(constants, 'ENABLE_NEW_STRUCTURE_EDITORS', True):
+            question_summary_dicts = self.get_json(
+                '%s/%s' %
```

Comment:
Please use a temporary value here instead, this is pretty tough to read.

Reply:
Done